src/components/SlateEditor/plugins/details/onBackspace.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
```

Comment:
Dei fleste av filene her er i details-pluginen og er retta mot den. Dersom tanken er at dette skal være meir generell kode for å håndtere delete og backspace i editoren må dette skrives om. 
Både tekst i ramme og fasitboks lar seg fjerne ved at du plasserer markøren i en tom boks og trykker backspace. Tanken bak denne funksjonen var vel at details skulle kunne la seg fjerne på samme måte gitt at summary er tom.